### PR TITLE
Initial Note Project

### PR DESCRIPTION
**Pull request by rahulcs754**
- Just added routes

### CODE REVIEW

**Review by nakulsharma15 — APPROVED** [review]
Everything Looks good. No changes suggested. Liked the way you structured the files

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (coldpigli) [inline public/index.html]: Change the title to the title of your App

**Review by coldpigli — APPROVED** [review]
Just one minor Suggestion. Rest all looks good

File: public/index.html (hunk 1 above)
Comment (rahulcs754) [reply public/index.html]: Thanks to pointing out this error. it will solve next pr

**Pull request merged**